habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -250,7 +250,8 @@ def observations_to_image(observation: Dict, info: Dict,
                 obs_k = obs_k.astype(np.uint8)
             if obs_k.shape[2] == 1:
                 obs_k = np.concatenate([obs_k for _ in range(3)], axis=2)
-            render_obs_images.append(obs_k)
+            if obs_k.shape[0] == 256 or obs_k.shape[0] == 512:
+                render_obs_images.append(obs_k)
```

Comment:
Could you please explain the changes here?

Reply:
In vip, I need to save the point coordinates in observations, and its dimension will conflict with the condition of video generation. So I modify this part to only append "real" image into video frames.